www/artists/dhall/stylesheet.css

```diff
@@ -0,0 +1,42 @@
+.links{
+  text-align: center;
+  list-style-type: none;
+  text-decoration: none;
+}
+}
```

Comment:
This end brace should not be here. It terminates a block that does not exist. Including it could potentially cause bugs. CircleCI doesn't validate css, but Codemy does, and so do I.